app/controller/grid/Grid.js

```diff
@@ -0,0 +1,328 @@
+/**
+ * This class is the controller for the cpsi mapview WFS
+ * generic grid class
+ *
+ */
+Ext.define('CpsiMapview.controller.grid.Grid', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.cmv_grid',
+
+    requires: [
+        'Ext.menu.Menu',
+        'Ext.grid.filters.Filters',
+        'BasiGX.util.Layer',
+        'GeoExt.util.OGCFilter'
+    ],
+
+    /**
+    * The currently active spatial filter for the layer.
+    *
+    * @cfg {Ext.util.Filter} spatialFilter
+    */
+    spatialFilter: null,
+
+    /**
+     * Fire a onRowDblClick to be handled by the parent application
+     */
+    onRowDblClick: function (sender, record) {
+        this.fireEvent('onRowDblClick', sender, record);
```

Comment:
Is it really necessary to forward the `rowdblclick` event of the grid?

Reply:
In a child application I used this approach, but I guess I can listen to the event directly from a view without needing to forward anything. Will remove from here.